mesh/mesh.cpp

```diff
@@ -9085,7 +9085,7 @@ void Mesh::NonconformingRefinement(const Array<Refinement> &refinements,
 double Mesh::AggregateError(const Array<double> &elem_error,
                             const int *fine, int nfine, int op)
 {
-   double error = 0.0;
+   double error = elem_error[fine[0]];
```

Comment:
This is a somewhat orthogonal bugfix. When using op=0, the minimum operator for aggregation, initializing to 0 means you never get anything but 0 as the final result (assuming you aren't supplying negative errors).

Reply:
This is now incorrect for the case `op==1` since the loop below starts from 0.

Reply:
I think this is a github bug or limitation. The loop starts from 1 now.